src/Microsoft.Health.SqlServer/Features/Schema/Manager/SqlSchemaManager.cs

```diff
@@ -65,7 +65,7 @@ public virtual async Task ApplySchema(MutuallyExclusiveType type, bool force = f
 
             int retryCountForHttpRequestException = 18;
 
-            var availableVersions = await Policy.Handle<HttpRequestException>()
+            _ = await Policy.Handle<HttpRequestException>()
```

Comment:
It doesn't look like we actually needed to save this value as it was immediately re-assigned